Gemfile

```diff
@@ -45,6 +45,8 @@ gem "bootsnap", require: false
 # Use Active Storage variants [https://guides.rubyonrails.org/active_storage_overview.html#transforming-images]
 # gem 'image_processing', '~> 1.2'
 
+gem "flipflop" # Feature flag
```

Comment:
Is the comment useful? If it's some type of category, maybe use something more generic like "infrastructure/deployment/...".
Good work